Dockerfile.aarch64

```diff
@@ -23,7 +23,7 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:trusty
+FROM aarch64/ubuntu:vivid
```

Comment:
switching to jessie fails on my board to build. Does it work on your arm64 board?

ubuntu@ubuntu:~/docker_latest/docker$ sudo docker version
Client:
 Version:      1.10.0-dev
 API version:  1.22
 Go version:   go1.5.2
 Built:        Thu Dec 17 09:07:33 2015
 OS/Arch:      linux/arm64
 Experimental: true

Server:
 Version:      1.10.0-dev
 API version:  1.22
 Go version:   go1.5.2
 Git commit:   66581cd-dirty
 Built:        Thu Dec 17 09:07:33 2015
 OS/Arch:      linux/arm64
 Experimental: true
ubuntu@ubuntu:~/docker_latest/docker$ sudo make 
docker build  -t "docker-dev:master" -f "Dockerfile.aarch64" .
Sending build context to Docker daemon 118.7 MB
Step 1 : FROM debian:jessie
 ---> 9a02f494bef8
Step 2 : RUN apt-get update && apt-get install -y       apparmor        aufs-tools      automake        bash-completion         s
 ---> Running in 0e256128c870
exec format error
Cannot start container 0e256128c8704505b866f6e2918d4b45a9f9a29bbb79c715afe2caef34f8140c: [9] System error: exec format error
Makefile:93: recipe for target 'build' failed
make: *** [build] Error 1
ubuntu@ubuntu:~/docker_latest/docker$